Catappult/appcoins-wallet-android
# [APPC-4153] Change currencies when not supported on TopUp and IAP

**What does this PR do?**

   Change currencies to supported one when not supported on TopUp and IAP

**Database changed?**

   No

**How should this be manually tested?**

 With VPN to any country that has non-supported currencies, like Brazil, select one of the payment methods with currency not support and verify whether currencies change correctly on TopUp and IAP.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4153](https://aptoide.atlassian.net/browse/APPC-4153)

**Code Review Checklist**



[APPC-4153]: https://aptoide.atlassian.net/browse/APPC-4153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
